File: pages/home_page.py
from selenium.webdriver.common.by import By

class HomePage:
    def __init__(self, driver):
        self.driver = driver
        self.checkbox_link = (By.LINK_TEXT, "Checkboxes")
        self.file_upload_link = (By.LINK_TEXT, "File Upload")

    def go_to_checkboxes_page(self):
        self.driver.find_element(*self.checkbox_link).click()

    def go_to_file_upload_page(self):
        self.driver.find_element(*self.file_upload_link).click()

File: src/test/java/com/capstone/selenium/pages/HomePage.java
package com.capstone.selenium.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class HomePage {
    WebDriver driver;

    By abTestLink = By.linkText("A/B Testing");
    By dropdownLink = By.linkText("Dropdown");
    By framesLink = By.linkText("Frames");

    public HomePage(WebDriver driver) {
        this.driver = driver;
    }

    public void clickABTesting() {
        driver.findElement(abTestLink).click();
    }

    public void clickDropdown() {
        driver.findElement(dropdownLink).click();
    }

    public void clickFrames() {
        driver.findElement(framesLink).click();
    }
}
